### ISSUE TITLE
BUG: Fields of parent classes are not logged

### ISSUE THREAD

**Issue by LenarBad** [LenarBad/easylog#31]
Make sure we don't miss any field (public, private and inherited)

**Comment by LenarBad** [comment]
Reaserch:
Method to log
```
    public MySubClass myMethod(MySubClass mySubClass) {
        return mySubClass;
    }
```
MySubClass
```
public class MySubClass extends MyClass {

    public String mySubClassPublicField;
    public String mySubClassPrivateField;
    public String mySubClassProtectedField;

    public MySubClass(
            String mySubClassPublicField,
            String mySubClassPrivateField,
            String mySubClassProtectedField,
            String myClassPublicField,
            String myClassPrivateField,
            String myClassProtectedField) {
        super(myClassPublicField, myClassPrivateField, myClassProtectedField);
        this.mySubClassPublicField = mySubClassPublicField;
        this.mySubClassPrivateField = mySubClassPrivateField;
        this.mySubClassProtectedField = mySubClassProtectedField;

    }
}
```
that has a parent MyClass
```
public class MyClass {

    public String myClassPublicField;
    private String myClassPrivateField;
    protected String myClassProtectedField;

    public MyClass(String myClassPublicField, String myClassPrivateField, String myClassProtectedField) {
        this.myClassPublicField = myClassPublicField;
        this.myClassPrivateField = myClassPrivateField;
        this.myClassProtectedField = myClassProtectedField;
    }

}
```

Results:
```
10:55:28.262 [main] INFO  UneasyLogger - 
CLIENT
-> public MySubClass Client.myMethod(MySubClass mySubClass)
{
  "mySubClass": {
    "mySubClassPublicFields": "mySubClassPublicFields",
    "mySubClassPrivateFields": "mySubClassPrivateFields",
    "mySubClassProtectedFields": "mySubClassProtectedFields"
  }
}

10:55:28.272 [main] INFO  UneasyLogger - 
Execution/Response time:  10ms
CLIENT
<- MySubClass Client.myMethod(MySubClass mySubClass)
{
  "mySubClassPublicFields": "mySubClassPublicFields",
  "mySubClassPrivateFields": "mySubClassPrivateFields",
  "mySubClassProtectedFields": "mySubClassProtectedFields"
}
```

**Conclusions**:
This is a bug.
We log all fields of sub classes (```public```, ```private```, ```protected```)
We log none of fields of a parent class.

**Comment by LenarBad** [comment]
Released in 1.1.0
